handler/category_dimension_import_test.go

```diff
@@ -126,6 +129,37 @@ func TestHandle(t *testing.T) {
 				So(sleepRandomCalls, ShouldHaveLength, 0)
 			})
 		})
+
+		Convey("When Handle is successfully triggered with only Geography codelists", func(c C) {
+			e := &event.CategoryDimensionImport{
```

Comment:
Instead of defining this event here perhaps it could be defined in the list of vars at the top and named something like testEventGeog

Reply:
This has been addressed on this PR: https://github.com/ONSdigital/dp-import-cantabular-dimension-options/pull/46